lib/hexpm/short_urls/short_url.ex

```diff
@@ -0,0 +1,58 @@
+defmodule Hexpm.ShortURLs.ShortURL do
+  use Hexpm.Schema
+  alias Hexpm.ShortURLs
+  alias Hexpm.ShortURLs.ShortURL
+
+  schema "short_urls" do
+    field :url, :string
+    field :short_code, :string
+
+    timestamps()
```

Comment:
this is pretty pedantic but since they are immutable we can do this:

```suggestion
    timestamps(updated_at: false)
```